vtex-sites/starter.store
# chore: WebOps Evergreen + Vercel

### depends on https://github.com/vtex/faststore/pull/1618

## What's the purpose of this pull request?

This PR aims to use our CLI to make changes that permit this repo works using WebOps and Vercel.

## How does it work?

It uses the Faststore CLI with these changes https://github.com/vtex/faststore/pull/1618 to make the build, deploy, SonarQube, LH, and cypress work as expected in Webops and Vercel.

It also adds some dependencies and configs that were necessary during the way.


## How to test it?

- you can see the preview links generated on this PR: vercel and webops.
- you can run locally by running `yarn build` in faststore repo package, adding the `packages/cli/bin` folder to your `$PATH`, and running `run dev`. ([how to guide](https://www.cyberciti.biz/faq/appleosx-bash-unix-change-set-path-environment-variable/))
- you can install `@faststore/cli` globally and test the `faststore` commands (build, dev, start) after the cli PR is merged.
```cmd
npm install -g @faststore/cli
```

## Faststore CLI PR
https://github.com/vtex/faststore/pull/1618

## References

https://www.cyberciti.biz/faq/appleosx-bash-unix-change-set-path-environment-variable
